parm/atm/obs/config/satwind.yaml

```diff
@@ -37,202 +445,440 @@ obs filters:
-#   action:
-#     name: reject
-  # GOES IR (245), EUMET IR (253), JMA IR (252) reject when pressure between 400 and 800 mb.
+#
+# preQC (read_satwnd) criteria
+#
+# EUMETSAT winds: satelliteIdentifer [50–79] (>49, <80)
+  # Reject obs with satelliteZenithAngle > 68 deg
+  # NOT EXPLICITLY CLEARED: No obs in this range in file, so 0 Bounds Check rejects (which is correct) but essentially untested
   - filter: Bounds Check
     filter variables:
     - name: eastward_wind
     - name: northward_wind
+    where:
+    - variable: MetaData/satelliteIdentifier
+      minvalue: 49
+      maxvalue: 80
     test variables:
-    - name: MetaData/air_pressure
-    minvalue: 40000
-    maxvalue: 80000
+    - name: MetaData/satelliteZenithAngle
+    maxvalue: 68.
+    action:
+      name: reject
+  # Reject obs with windComputationMethod = 5 (clear-sky WV AMV)
+  # CLEARED
+  - filter: Bounds Check
+    filter variables:
+    - name: eastward_wind
+    - name: northward_wind
     where:
-    - variable:
-        name: ObsType/eastward_wind
-      is_in: 245, 252, 253
+    - variable: MetaData/satelliteIdentifier
+      minvalue: 49
+      maxvalue: 80
+    test variables:
+    - name: MetaData/windComputationMethod
+    maxvalue: 4.
     action:
       name: reject
-  # GOES WV (246, 250, 254), reject when pressure greater than 400 mb.
+  # Reject obs with qualityInformationWithoutForecast < 85.
+  # CLEARED
   - filter: Bounds Check
     filter variables:
     - name: eastward_wind
     - name: northward_wind
+    where:
+    - variable: MetaData/satelliteIdentifier
+      minvalue: 49
+      maxvalue: 80
     test variables:
-    - name: MetaData/air_pressure
-    maxvalue: 40000
+    - name: MetaData/qualityInformationWithoutForecast 
+    minvalue: 85.
+    action:
+      name: reject
+# JMA: satelliteIdentifier [100–199] (>99, <200)
+  # Reject obs with satelliteZenithAngle > 68 deg
+  # NOT EXPLICITLY CLEARED: No obs in this range in file, so 0 Bounds Check rejects (which is correct) but essentially untested
+  - filter: Bounds Check
+    filter variables:
+    - name: eastward_wind
+    - name: northward_wind
     where:
-    - variable:
-        name: ObsType/eastward_wind
-      is_in: 246, 250, 254
+    - variable: MetaData/satelliteIdentifier
+      minvalue: 99
+      maxvalue: 200
+    test variables:
+    - name: MetaData/satelliteZenithAngle
+    maxvalue: 68.
     action:
       name: reject
-  # EUMET (242) and JMA (243) vis, reject when pressure less than 700 mb.
+  # Reject obs with windComputationMethod = 5 (clear-sky WV AMV)
+  # NOT EXPLICITLY CLEARED: No obs in this range in file, so 0 Bounds Check rejects (which is correct) but essentially untested
+  - filter: Bounds Check
+    filter variables:
+    - name: eastward_wind
+    - name: northward_wind
+    where:
+    - variable: MetaData/satelliteIdentifier
+      minvalue: 99
+      maxvalue: 200
+    test variables:
+    - name: MetaData/windComputationMethod
+    maxvalue: 4.
```

Comment:
Some whole number values appear to be integers (no "." after the number) while others appear to be floats (with a "." after the number). Does it matter which one they are? Regardless, do we want to standardize?

Reply:
I think it's worthwhile to consider a standardization, particularly since some `where` functions like `is_in` and `is_not_in` only work with integers. The example with `MetaData/windComputationMethod`, for example, is a flag that should probably be an integer, but is being stored as a float. In this case I'm trying to be explicit about the variable as a float in the YAML file – if it were an integer, I would probably use `is_not_in: 5` instead of `maxvalue: 4.`.

Reply:
Based on our group conversation on Wednesday, I think we want to adhere to a convention where a float is `4.0` or `4.` and an integer is `4`.  So that likely means this PR needs updated to adhere to this everywhere, right @BrettHoover-NOAA ?

Reply:
@CoryMartin-NOAA Yeah, that seems to be the consensus. I've converted this PR to a draft and I'll revise.

Reply:
Updated PR now asserts any float `x` as `x.`